implementations/micrometer-registry-elastic/src/main/java/io/micrometer/elastic/ElasticMeterRegistry.java

```diff
@@ -123,23 +123,27 @@ protected void publish() {
 
         for (List<Meter> batch : MeterPartition.partition(this, config.batchSize())) {
             try {
+                String jsonContent = batch.stream()
+                    .map(m -> m.match(
+                        this::writeGauge,
+                        this::writeCounter,
+                        this::writeTimer,
+                        this::writeSummary,
+                        this::writeLongTaskTimer,
+                        this::writeTimeGauge,
+                        this::writeFunctionCounter,
+                        this::writeFunctionTimer,
+                        this::writeMeter))
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .collect(joining("\n", "", "\n"));
+
+                logger.trace("Sending metrics json to elastic: {}", jsonContent);
```

Comment:
Could we avoid the trace call here?

Reply:
Sure, I just though it is very helpful. With this I was able to spot the bug. We have the same trace calls/pattern i.e. in https://github.com/micrometer-metrics/micrometer/blob/2a59cd638739edcb2cb36ccc72e0a5520c158f97/implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogMeterRegistry.java#L117-L131 so I thought it makes sense to add it here as well.

Just let me know if to remove or not. Performance wise it should not add any overhead when tracing is disabled.